keda/values.yaml

```diff
@@ -3,8 +3,8 @@
 # Declare variables to be passed into your templates.
 
 image:
```

Comment:
This was already updated, can you rebase please?

Reply:
Should be resolved!